client/package.json

```diff
@@ -15,6 +15,7 @@
     "react": "^18",
     "react-chartjs-2": "^5.2.0",
     "react-dom": "^18",
+    "react-icons": "^4.12.0",
```

Comment:
react-icons might increase the bundle size a lot - so I would suggest just to use svg since we only have two icons.

It would just be replacing the icons with a svg component wrapped by a button with same functionalities. I've added a couple of assets for the same to this repo, the color can be changed by changing the fill property.